resources/sync_gateway_configs/listener_tests/multiple_sync_gateways_cc.json

```diff
@@ -21,7 +21,7 @@
             {{ autoimport }}
             {{ xattrs }}
             {{ no_conflicts }}
-            {{ revs_limit }}
+            "revs_limit": 25,
```

Comment:
why are we hardcoding values here?

Reply:
this files are required only for specific tests which needs specific revs limit. As it is multiple sgs, I have a problem of overriding cluster config with revision limit for temporary file as cluster configs temporary files gets override each other and both sync-gateways are having same revision limit at the end which I see issue and change it hard coded with different revision limit